apps/tsconfig.json

```diff
@@ -1,33 +1,15 @@
+// This tsconfig is used by your code editor, such as VS Code. 
+// It extends tsconfig.build.json, used for building apps.
 {
```

Comment:
I'd love to see a comment here that indicates what kinds of config should be in `tsconfig.build.json` vs in `tsconfig.json`. Right now its pretty clear "oh, compilerOptions are all in the other file", but over time, people forget the what and why, and two files often end up getting a confusing mishmash of configs one place or another.

Is there a sane way to articulate what should be shared with webpack ts-loader config and what shouldn't?

Reply:
I took a stab at this, let me know if it's more clear!